packages/react/package.json

```diff
@@ -12,6 +12,10 @@
       "import": "./dist/esm/internal.js",
       "require": "./dist/internal.js"
     },
+    "./internal/primitive": {
```

Comment:
```suggestion
    "./internal/primitives-catalog": {
```

Reply:
Should we be consistent with `primitive` ? The json version is named `primitive`. I think this will cause confusion but they are the same in different forms.